Common/GL/GLInterface/GLInterface.cpp

```diff
@@ -7,7 +7,7 @@
 
 #ifdef __ANDROID__
 #include "Common/GL/GLInterface/EGLAndroid.h"
-#elif PPSSPP_PLATFORM(SWITCH)
+#elif defined(HAVE_LIBNX)
```

Comment:
Is this intentional?  Any specific reason not to use `PPSSPP_PLATFORM(SWITCH)`?

-[Unknown]